src/main/java/in/twizmwaz/cardinal/module/modules/teamPicker/TeamPicker.java

```diff
@@ -52,12 +55,20 @@ public Inventory getTeamPicker(Player player) {
                 totalPlayers += team.size();
             }
         }
```

Comment:
@iEli2tyree011 It shouldn't do that. As when you click with the hat, it creates a new inventory and shows it the player.